tests/InputFiles/GeneralizedHarmonic/KerrSchild.yaml

```diff
@@ -97,6 +97,8 @@ EventsAndTriggers:
           - PointwiseL2Norm(ThreeIndexConstraint)
           - PointwiseL2Norm(FourIndexConstraint)
         InterpolateToMesh: None
+        CoordinatesFloatingPointType: Float
+        FloatingPointTypes: [Float]
```

Comment:
I have nothing against the proposed names, though I can't really allocate the time to changing the names since updating them requires changing a bunch of input files.